extensions/amp-story-page-attachment/0.1/amp-story-draggable-drawer.js

```diff
@@ -517,8 +517,12 @@ export class DraggableDrawer extends AMP.BaseElement {
           return;
         }
         this.state = DrawerState.DRAGGING_TO_OPEN;
+        if (this.containerEl.hasAttribute('hidden')) {
+          this.mutateElement(() => {
+            toggle(dev().assertElement(this.containerEl), true);
```

Comment:
I'm sure that if `this.containerEl.hasAttribute('hidden')` is true, `dev.assertElement` is not needed. If we don't know whether containerEl is created, then you can just check `this.containerEl?.hasAttribute("hidden")` (I don't remember when this happens, so it might not be necessary)